### DISCUSSION TITLE
Disabling scroll bars in ListView

### DISCUSSION THREAD

**Discussion by freelon** [slint-ui/slint#2501] [General]
Hi,

I'm running Slint on Linux with all default-settings. My ListView always has a horizontal and vertical scrollbar in the running app (not in the preview in VSCode, though). I've seen that in the widget gallery the ListView doesn't have the unnecessary scroll bars, but there isn't a flag for that.

Is there something I am missing? Or does this depend on the renderer?

*appwindow.slint:*

    ...
    VerticalBox { 
        LineEdit { 
            placeholder-text: "service name";
            accepted => {
                root.start-service(self.text);
                self.text = "";
            }
         }
         ListView { 
            width: 150px;
            height: 260px;
            for service in services : HorizontalBox {
                Text {
                    text: service.name;
                }
                Button {
                    text: "Stop";
                    clicked => {
                        stop-service(service.name);
                    }
                }
            }
          }
     }
    ...

![image](https://user-images.githubusercontent.com/2052561/230388832-5812f4c8-891c-472a-ac9d-8d5bd0e441f7.png)

**Comment by Farmbot26** [comment]
I am experiencing the same thing. No amount of stretch or width settings hide the unnecessary scroll bars.

**Comment by ogoffart** [comment]
Currently it is not possible to hide the scrollbar, we would need to fix that.
May also relates to https://github.com/slint-ui/slint/issues/988 and https://github.com/slint-ui/slint/issues/824

**Reply by jerrygou007** [reply]
have the same problem,
is it fixed?

**Reply by bjorn** [reply]
No. For a more specific issue to watch, see #3416.

There has been some improvement for users of the "Fluent" style, since the scrollbars there are overlay scrollbars now, which never take up any space and are invisible when they are not needed.
